### DISCUSSION TITLE
GitHub Universe Virtual Guide – share your setup with us!

### DISCUSSION THREAD

**Discussion by ebndev** [community/community#142516] [Other Feature Feedback, Questions, & Ideas]
![Universe'24 - Virtual Guide (1)](https://github.com/user-attachments/assets/aeb14a8c-28f2-4b43-b256-4463d30dd872)

Attending GitHub Universe virtually this year? We’d love to see where you’re joining from! `Share a photo of your desk setup in the comments below`, and let’s get ready for an awesome virtual experience. ⭐

### Plan Ahead
Building out your schedule is key to making the most of the event. Browse through the [GitHub Universe session catalog](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog?tab.day=20241029) or explore the curated virtual agendas to pick the sessions you’re most excited about. 

### Create a Comfortable Environment
A comfortable workspace can make all the difference— whether it’s a laptop on your kitchen table, a decked out office with LED lights and multiple monitors, or somewhere in between. Be sure to have snacks and in-between session fuel! 🍬 Show us your go-to hunger warrior ⚔️

### Engage with the Community
Getting involved is easy, even remotely! Join the conversations through live chat during sessions, and check if any virtual happy hours or social events are happening. We’re considering a “Virtual Lounge” post in Discussions—**would you join in?**

### Take Breaks & Avoid Burnout
Don’t forget to stretch, hydrate, and take breaks to avoid screen fatigue. Short mindfulness exercises or stepping away from your desk can work wonders. What are some of the ways you plan to care for yourself while joining in virtually? 

### Get the Most Out of Your Sessions
We’ve got a lot of amazing talks and speakers on the agenda! Stay present by closing unnecessary tabs and focusing on each session. Use digital tools for note-taking, and create actionable items as you go. You can help us make the GitHub experience even better by reviewing the changelogs, contributing to discussions, and giving your feedback.

Do you have any tips or ideas for making your virtual experience special? Let’s make it a productive and fun GitHub Universe!

_Please note: any off-topic photos or inappropriate content may result in a temporary or indefinite block from the Community. Thanks for understanding._

**Comment by ebndev** [comment]
Not my setup but this would be a dream ☁️ 

![image](https://github.com/user-attachments/assets/aeca14b7-9131-4cdd-9737-24fcdb7ece43)

**Reply by MasterInQuestion** [reply]
And sincerely, I deem the color LED lights somewhat... garish.

**Comment by MasterInQuestion** [comment]
Scarcely use Desktop nowadays...
    (mostly Mobile)

**Reply by ebndev** [reply]
Oh, that's interesting you're able to do everything from Mobile! Do you run into any issues?

**Reply by MasterInQuestion** [reply]
Countless many.
    Some not easily solvable (and not yet solved).
    But anyway crunched on... as is everything else.
